package/src/store/apis/deleteMember.ts

```diff
@@ -5,12 +5,12 @@ import { SqliteClient } from '../SqliteClient';
 
 export const deleteMember = async ({
   cid,
-  flush = true,
+  execute = true,
```

Comment:
What would the `execute` false in these DB APIs do? Seems like this won't execute the query right?

Reply:
Correct ! It will aggregate the queries in a prepared format that can then be put into `offlineDb.executeSqlBatch(queries)` so that they're executed in a batch-like fashion. So, each API always returns the queries but `execute` controls whether they're executed or not.

Reply:
Yep. It's mainly useful when you want to do a composition of multiple queries within a single scope, you want to:

- Execute them in a batch for performance reasons
- Make sure they're wrapped within a transaction for write lock purposes

both of which are covered by batch execution.